network-protection/network-protection-api/src/main/java/com/duckduckgo/networkprotection/api/NetworkProtectionWaitlist.kt

```diff
@@ -37,6 +37,6 @@ interface NetworkProtectionWaitlist {
         object NotUnlocked : NetPWaitlistState()
         object JoinedWaitlist : NetPWaitlistState()
         object PendingInviteCode : NetPWaitlistState()
-        object InBeta : NetPWaitlistState()
+        data class InBeta(val termsAccepted: Boolean) : NetPWaitlistState()
```

Comment:
Addresses [this comment](https://app.asana.com/0/0/1205230811122378/1205255564162048/f)